tests/test_data_requests.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+
+from app.main.data_requests import calculate_days_remaining
+from config import Config
+
+
+def test_calculate_future_date(flask_test_client):
+    mock_current_date = datetime.strptime("01/09/2023", "%d/%m/%Y").date()
+
+    Config.SUBMIT_DEADLINE = "15/11/2023"
+    days_remaining = calculate_days_remaining(mock_current_date)
+    assert days_remaining == 75
+
+
+# TODO consider desired behaviour if deadline date is in the past
```

Comment:
If I was you, I'd add another test that also calls this function as it's generally used in the code, ie let it use it's default. And just a basic assertion to check it actually works (ie assert you actually get an int for the return val)

Reply:
This also isn't implemented